src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1195,6 +1196,7 @@ public partial class LiveRunner : ILiveRunner, IDisposable
     {
         private IDictionary<string, IList> graphOutput;
         internal bool DSExecutionEngine = true;
+        internal long ExecutionTime = 0;
```

Comment:
I think if we make this a timespan it's less possible to get confused about units.